.gitignore

```diff
@@ -11,6 +11,7 @@
 /test/tmp/
 /test/version_tmp/
 /tmp/
+lib/gen/
```

Comment:
I think the `gen` is quite small they actually look good to me when reading the code there as well 🤔 . I feel like we could commit it as-is and reduce the need to generate them again with `pub build` again when it's on CI 🤔 

Would you guys agree to keep instead of ignore? 🤓